src/components/actor-card/actor-card.js

```diff
@@ -0,0 +1,27 @@
+import { Card, CardContent, CardMedia, Typography, CardActionArea } from '@mui/material';
+import actor from './actor.jpg';
+
+const ActorCard = () => {
+  return (
+    <Card sx={{ backgroundColor: '#151515', marginBottom: 2, minWidth: 150 }}>
```

Comment:
instead of these inline styles better try to write styles in scss files and bing them by className